app/views/users/show_follow.html.erb

```diff
@@ -17,7 +17,9 @@
         </tr>
       <% end %>
     </table>
-    <%= will_paginate @users, renderer: WillPaginate::ActionView::BootstrapLinkRenderer %>
+   <% if @pagy %> <%= raw pagy_bootstrap_nav @pagy %> 
```

Comment:
Hey @KarenEfereyan could you please separate this to two lines like so, 
```
<% if @pagy %>
  <%= raw pagy_bootstrap_nav @pagy %> 
```
This is mainly for readability purposes thanks

Reply:
I will. Thank you